Huizenjacht/src/main/java/be/pxl/huizenjacht/service/HouseService.java

```diff
@@ -1,47 +0,0 @@
-package be.pxl.huizenjacht.service;
-
-import be.pxl.huizenjacht.domain.House;
-import be.pxl.huizenjacht.domain.Status;
-import org.springframework.stereotype.Service;
-
-import java.util.*;
-
-@Service
-public class HouseService {
-    private List<House> houseList = new ArrayList<>();
-    private Map<String, House> houseMap = new HashMap<>();
-
-    public List<House> getHouses() {
-        //return houseList;
-        return houseMap.values().stream().toList();
-    }
-//    public Collection<House> mamaMia() {
-//        //return houseList;
-//        return houseMap.values();
-//    }
-
-
-    public void deleteHouse(String code) {
-        houseMap.remove(code);
-    }
-
-    public void addHouse(House house) {
-        houseMap.put(house.getCode(), house);
-    }
-
-    public void updateHouse(String code, House house) {

```

Comment:
// TODO: Throw an own HouseNotFoundException (custom unchecked exception) when there is no house with the given code